engine/samples/assets/json/page.md

```diff
@@ -0,0 +1,26 @@
+# Loading Serializable Assets {#examples-engine-assets-json}
+
+@brief Loading serializable assets from JSON.
+
+We'll use the following type as an example:
+
+@snippet assets/json/main.cpp Asset type
+
+We can make it serializable by implementing the following specializations:
```

Comment:
Explain what the `const char* name` argument is and how it should be used

Reply:
No use documenting/explaining serialization since we should get rid of it quickly.